tests/system/provisioning/vars/wazuh_agent.yml

```diff
@@ -15,4 +15,4 @@ wazuh_winagent_config_url: https://s3-us-west-1.amazonaws.com/packages-dev.wazuh
 wazuh_winagent_package_name: wazuh-agent-3.12.0-0.3319fimreworksqlite.msi
 
 wazuh_winagent_config:
-  md5: 598ae7999f9dfec77f960e994c218d52
```

Comment:
Leave `md5` parameter please, it won't be used as `check_md5` will be False but might serve as reference in case we decide to activate it

Reply:
Fixed in latest commit.